zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -603,7 +603,7 @@ paths:
           description: >
             The clock was successfully pinned to the specified time in epoch milliseconds.
         "400":
-          description: The required timestamp parameter is missing.
+          description: The required timestamp parameter is missing or if a negative timestamp value is provided.
```

Comment:
```suggestion
          description: The required timestamp parameter is missing or it is negative.
```

❌ Can you also add in `ClockPinRequest` schema, under `timestamp` field, the property `minimun: 0` ?

Reply:
Wording improved, added minimum prop for `ClockPinRequest#timestamp` field.